functions/jsonwebtoken.js

```diff
@@ -0,0 +1,33 @@
+require('dotenv').config();
+const jwt = require('jsonwebtoken');
+const getTokenFrom = require('./get-token-from');
+
+module.exports = {
+  generateAccesToken: (user) => {
+    if (!user.role || !user.id || !user.username) return null; // access denied, it returns null
+
+    const userForToken = {
+      username: user.username,
```

Comment:
As we don't have an `username` in User's model, I think we could use the `email` instead

Reply:
I'm sorry! I made a mistake, I'll change it in a minute.